config/ext_jenkins-weekly.yaml

```diff
@@ -0,0 +1,154 @@
+serviceAccount:
+  create: true
+  name: jenkins-controller
+serviceAccountAgent:
+  create: false
+rbac:
+  create: true
+  readSecrets: true
+persistence:
+  enabled: true
+  size: 50Gi
```

Comment:
What could be the minimal size here?

Reply:
depends what type of volume is in use, 4gb is the smallest ssd or 32 for standard hdd by the looks of it.

https://docs.microsoft.com/en-us/azure/virtual-machines/disks-types#standard-ssd-size